build.gradle

```diff
@@ -37,13 +36,20 @@ subprojects {
 	apply plugin: 'com.github.ben-manes.versions'
 	apply from: rootProject.file('gradle/publishing.gradle')
 
-
 	sourceCompatibility = "1.8"
 	targetCompatibility = "1.8"
 
 	repositories {
 		mavenLocal()
 		jcenter()
+
+		maven {
+			credentials {
+				username smartThingsUserName
+				password smartThingsPassword
+			}
+			url "https://smartthings.artifactoryonline.com/smartthings/libs-release-local"
```

Comment:
What is this needed for?  I don't think we want the OSS repos to pull from our internal artifactory.